src/ui/zcl_abapgit_gui_page_stage.clas.abap

```diff
@@ -135,6 +147,39 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_STAGE IMPLEMENTATION.
 
   ENDMETHOD.
 
+  METHOD find_transports.
+    DATA: li_cts_api TYPE REF TO zif_abapgit_cts_api,
+          ls_new     LIKE LINE OF rt_transports.
+    FIELD-SYMBOLS: <ls_local> LIKE LINE OF it_local,
+                   <ls_new>   LIKE LINE OF rt_transports.
+
+    li_cts_api = zcl_abapgit_factory=>get_cts_api( ).
+
+    TRY.
+        LOOP AT it_local ASSIGNING <ls_local> WHERE item IS NOT INITIAL.
+          IF <ls_local>-item-obj_type IS NOT INITIAL AND
+             <ls_local>-item-obj_name IS NOT INITIAL AND
+             <ls_local>-item-devclass IS NOT INITIAL AND
+             li_cts_api->is_chrec_possible_for_package( <ls_local>-item-devclass ) = abap_true AND
```

Comment:
I think we can safely assume that if the first package is not transportable, then nothing is transportable. can we RETURN if `is_chrec_possible_for_package = false`